components/plan-build/bin/public.bash

```diff
@@ -234,13 +236,12 @@ Aliases
   @             Alias for \`whereami\`.
   quit          Alias for \`exit\`.
   quit-program  Alias for \`exit-program\`.
-
 "
         ;;
-      *) eval $cmd;;
+      *) eval "$cmd";;
     esac
     # Increment our REPL command line count, cause that's helpful
-    replno=$((${replno}+1))
+    replno=$((replno + 1))
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2004#-is-unnecessary-on-arithmetic-variables